### PR TITLE
Implement a few mxomni functions

### PR DESCRIPTION
**Pull request by MishaProductions**
- Implemented MxOmni::Create (doesn't match probably due to inline constructors)
 - Implemented MxOmni::StartTimer and StopTimer
 - Improved accuracy of MxOmni::Notify
 As a result of implementing MxOmni::Create, the game no longer crashes with our lego1 when you run it.

### CODE REVIEW

File: LEGO1/mxeventmanager.cpp [hunk 1]
```diff
@@ -15,4 +16,11 @@ MxEventManager::~MxEventManager()
 // OFFSET: LEGO1 0x100c0450
 void MxEventManager::Init()
 {
-}
\ No newline at end of file
+}
+
+// OFFSET: LEGO1 0x100c04a0 STUB
+MxResult MxEventManager::vtable0x28(undefined4 p_unknown1, MxU8 p_unknown2)
```
Comment (foxtacles) [inline LEGO1/mxeventmanager.cpp:22]: `MxU8` should be `undefined`

File: LEGO1/mxomni.cpp [hunk 2]
```diff
@@ -51,7 +51,7 @@ void MxOmni::Init()
   m_timer = NULL;
   m_streamer = NULL;
   m_atomIdCounterSet = NULL;
-  m_unk64 = NULL;
+  m_TimerRunning = FALSE;
```
Comment (foxtacles) [inline LEGO1/mxomni.cpp:54]: Should be called `m_timerRunning` based on our style guide

**Comment by foxtacles** [comment]
Could you split this into smaller PRs that each focus on one thing at a time? This way it would be much easier to review. Thank you!

**Comment by MishaProductions** [comment]
Yeah I probably should have done that.

**Pull request closed without merging**